tests/contrib/tornado/test_stack_context.py

```diff
@@ -23,7 +23,7 @@ def test_stack_context(self):
         with TracerStackContext():
             ctx = self.tracer.context_provider.active()
 
-        assert ctx is not None
+        assert ctx is None
```

Comment:
wait, is this change correct? the comment above might not reflect this behavior properly?

> a TracerStackContext should automatically propagate a tracing context

or that... once we leave the scope of a `TracerStackContext()` the `ctx` is cleared out?

should we do:

```python
with TracerStackContext():
    ctx = self.tracer.context_provider.active()
    assert ctx is not None

assert ctx is None
```

? I think I might be misunderstanding what is changing, or what the expected behavior is?

Reply:
Ah yeah this test might be wrong for the moment. The tornado context management needs to be corrected/reworked to fit into the new model (one of the last things to implement).

Reply:
Ok so looks like this change is required. We changed the default context item for the `TracerStackContext` to be `None`.